src/test/java/com/easypost/AddressTest.java

```diff
@@ -171,7 +171,7 @@ public void testCreateAndVerify() throws EasyPostException {
         Map<String, Object> addressData = Fixtures.caAddress1();
 
         Address address = vcr.client.address.createAndVerify(addressData);
-
+        
```

Comment:
Remove this whitespace

Reply:
resolved, linter did not detect this for some reason :/